site-src/implementations/v1.3.md

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
It appears that these markdown files are not automatically updated by the `hack/mkdocs-generate-conformance.py` script. I suggest either removing them completely or updating the script to ensure they are generated correctly.

Reply:
Thanks for calling this out! Do you mind filing an issue for this?

Reply:
Sure, however I already fixed it in this PR.